medcat/1_create_model/create_cdb/create_umls_cdb.py

```diff
@@ -14,8 +22,8 @@
     os.makedirs('models')
     print("Creating a 'models' folder to store model")
 
-model_dir = './models/'
-output_cdb = model_dir + f"{release}_UMLS_cdb.dat"
+model_dir = os.path.join(BASE_PATH, "models", "cdb")
+output_cdb = os.path.join(model_dir, f"{release}_UMLS_cdb.dat")
 csv = pd.read_csv(csv_path)
```

Comment:
The csv_path should be changed to `EXPECTED_CSV_PATH` + the inputted `csv_path`. i.e Same as the create_cdb.py